runtime/coverage.go

```diff
@@ -230,6 +230,17 @@ func (r *CoverageReport) CoveredStatementsPercentage() string {
 	return fmt.Sprintf("Coverage: %v of statements", percentage)
 }
 
+// Reset flushes the collected coverage information for all locations
+// and inspected programs. Excluded locations remain intact.
+func (r *CoverageReport) Reset() {
+	for location, _ := range r.Coverage { // nolint:maprange
+		delete(r.Coverage, location)
+	}
+	for program, _ := range r.Programs { // nolint:maprange
+		delete(r.Programs, program)
+	}
```

Comment:
```suggestion
	for location := range r.Coverage { // nolint:maprange
		delete(r.Coverage, location)
	}
	for program := range r.Programs { // nolint:maprange
		delete(r.Programs, program)
	}
```

Reply:
Rookie mistakes :joy_cat: 
Fixed and rebased, to resolve the conflicts from the previous merge!